### PR TITLE
Estute/add hockey app support

### PR DESCRIPTION
**Pull request by estute**
@benpatterson @jzoldak 

+@clee  I will ask you to test the build script once I merge this

### CODE REVIEW

**Comment by clee** [comment]
Guys. I'm not @benjilee. C'mon.

File: mobileApp/jobs/buildWhiteLabelAndroidApps.groovy [hunk 1]
```diff
@@ -80,11 +85,13 @@ secretMap.each { jobConfigs ->
                     if (!jobConfig['public'].toBoolean()) {
                         credentials(jobConfig['gitCredential'])
                     }
-                    refspec('+refs/heads/master:refs/remotes/origin/master')
                 }
                 branch('\${HASH}')
                 browser()
                 extensions {
+                    submoduleOptions{
+                        disable(false)
```
Comment (benpatterson) [inline mobileApp/jobs/buildWhiteLabelAndroidApps.groovy]: Could you put a comment on this line to explain?

File: src/test/groovy/mobileApp/buildWhiteLabelAndroidAppsSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        setup:
+        String secretPath = baseSecretPath + '/non-existent-file.yml'
+        jm =  loadSecret(secretVar, secretPath)
+        loader = new DslScriptLoader(jm)
+
+        when:
+        GeneratedItems generatedItems = loader.runScript(dslScript.text)
+
+        then:
+        generatedItems.jobs.size() == 0
```
Comment (benpatterson) [inline src/test/groovy/mobileApp/buildWhiteLabelAndroidAppsSpec.groovy]: No exceptions in this case?

File: src/test/groovy/mobileApp/buildWhiteLabelAndroidAppsSpec.groovy [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    }
+
+    /**
+    * Using an invalid yaml secret, ensure that the appropriate exceptions are thrown, and
```
Comment (benpatterson) [inline src/test/groovy/mobileApp/buildWhiteLabelAndroidAppsSpec.groovy]: I don't see any exceptions thrown here in this method?

File: src/test/groovy/mobileApp/buildWhiteLabelAndroidAppsSpec.groovy [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+        setup:
+        String secretPath = baseSecretPath + '/build-android-secret.yml'
+        jm =  loadSecret(secretVar, secretPath)
+        loader = new DslScriptLoader(jm)
+
+        when:
+        loader.runScript(dslScript.text)
+
+        then:
+        noExceptionThrown()
```
Comment (benpatterson) [inline src/test/groovy/mobileApp/buildWhiteLabelAndroidAppsSpec.groovy]: Should you also assert that a job was created?

**Comment by benpatterson** [comment]
I had some questions but :+1: 

FWIW I wonder if you're getting the right hints from GitHub when you (at)mention someone. Our Chris and Jay should auto-populate if you start typing (at) + full-name of the person. That's what I do anyway. HTH

**Pull request merged**